record/record.app.js

```diff
@@ -48,6 +48,7 @@
             errorData = {};
         $rootScope.displayReady = false;
         $rootScope.showSpinner = false; // this property is set from common modules for controlling the spinner at a global level that is out of the scope of the app
+        $rootScope.loading = true;
```

Comment:
I'm not sure if this is important or not. But with this change, we're showing both `loading..`  and spinner while the main entity is being initialized. We might want to change the condition for showing `#rt-loading` to be based on both `loading` and `displayReady`.